x-pack/plugins/timelines/public/hooks/use_status_bulk_action_items.tsx

```diff
@@ -122,42 +124,63 @@ export const useStatusBulkActionItems = ({
   );
 
   const items = useMemo(() => {
-    const actionItems = [];
-    if (currentStatus !== FILTER_OPEN) {
-      actionItems.push(
-        <EuiContextMenuItem
-          key="open"
-          data-test-subj="open-alert-status"
-          onClick={() => onClickUpdate(FILTER_OPEN)}
-        >
-          {i18n.BULK_ACTION_OPEN_SELECTED}
-        </EuiContextMenuItem>
-      );
-    }
-    if (currentStatus !== FILTER_ACKNOWLEDGED) {
-      actionItems.push(
-        <EuiContextMenuItem
-          key="acknowledge"
-          data-test-subj="acknowledged-alert-status"
-          onClick={() => onClickUpdate(FILTER_ACKNOWLEDGED)}
-        >
-          {i18n.BULK_ACTION_ACKNOWLEDGED_SELECTED}
-        </EuiContextMenuItem>
-      );
-    }
-    if (currentStatus !== FILTER_CLOSED) {
-      actionItems.push(
-        <EuiContextMenuItem
-          key="close"
-          data-test-subj="close-alert-status"
-          onClick={() => onClickUpdate(FILTER_CLOSED)}
-        >
-          {i18n.BULK_ACTION_CLOSE_SELECTED}
-        </EuiContextMenuItem>
-      );
+    const actionItems: JSX.Element[] = [];
+    if (showAlertStatusActions) {
+      if (currentStatus !== FILTER_OPEN) {
+        actionItems.push(
+          <EuiContextMenuItem
+            key="open"
+            data-test-subj="open-alert-status"
+            onClick={() => onClickUpdate(FILTER_OPEN)}
+          >
+            {i18n.BULK_ACTION_OPEN_SELECTED}
+          </EuiContextMenuItem>
+        );
+      }
+      if (currentStatus !== FILTER_ACKNOWLEDGED) {
+        actionItems.push(
+          <EuiContextMenuItem
+            key="acknowledge"
+            data-test-subj="acknowledged-alert-status"
+            onClick={() => onClickUpdate(FILTER_ACKNOWLEDGED)}
+          >
+            {i18n.BULK_ACTION_ACKNOWLEDGED_SELECTED}
+          </EuiContextMenuItem>
+        );
+      }
+      if (currentStatus !== FILTER_CLOSED) {
+        actionItems.push(
+          <EuiContextMenuItem
+            key="close"
+            data-test-subj="close-alert-status"
+            onClick={() => onClickUpdate(FILTER_CLOSED)}
+          >
+            {i18n.BULK_ACTION_CLOSE_SELECTED}
+          </EuiContextMenuItem>
+        );
+      }
     }
-    return actionItems;
-  }, [currentStatus, onClickUpdate]);
+
+    const additionalItems = customBulkActions
+      ? customBulkActions.reduce<JSX.Element[]>((acc, action) => {
```

Comment:
maybe a `map` would be more appropriate?

Reply:
nope, just readability

Reply:
I don't thin it makes it less or more readable, just different code.